toolchain/gcc/common.mk

```diff
@@ -123,15 +123,12 @@ GCC_CONFIGURE:= \
 		--with-gmp=$(TOPDIR)/staging_dir/host \
 		--with-mpfr=$(TOPDIR)/staging_dir/host \
 		--with-mpc=$(TOPDIR)/staging_dir/host \
-		--disable-decimal-float
+		--disable-decimal-float \
+		--with-diagnostics-color=auto-if-env
```

Comment:
@diizzyy How is this related to this PR?

Reply:
https://github.com/openwrt/openwrt/pull/1848/files#diff-2d94c69a28fa3c28c325f985e624a9d6L131 ?